api/db/changelog/db.changelog-210-add-one-time-code.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I thought it was a good idea a few weeks ago, but it's definitely too complex for now. I removed the `purpose` field 👍

Reply:
I'm not sure what you mean by "user root table." Does that mean all tables that are children of the user table? I would probably use the name `new_user_satisfiction_survey_one_time_code` since the survey is specifically for new users.

Reply:
user rooted tables are user_access_module, user_access_tier, user_recent_resource, etc. 
`new_user_satisfiction_survey_one_time_code` SG!

Reply:
Done 👍